go/glide/glide.go

```diff
@@ -1 +1,82 @@
+/**
+ * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+ */
 package glide
+
+/*
```

Comment:
Can we remove that comment?

Reply:
Nope. It's actually read by cgo and is required to work with it.